embark-ui/src/components/ContractTransactions.js

```diff
@@ -115,21 +116,22 @@ class ContractTransactions extends React.Component {
                   <th>Gas Used</th>
                   <th>Block number</th>
                   <th>Status</th>
-                  <th>Transaction hash</th>
+                  <th>Result</th>
                 </tr>
               </thead>
               <tbody>
                 {
                   this.dataToDisplay().map((log, index) => {
                     return (
-                      <tr key={'log-' + index}>
-                        <td><DebugButton forceDebuggable transaction={{hash: log.transactionHash}}/></td>
-                        <td>{`${log.name}.${log.functionName}(${log.paramString})`}</td>
-                        <td>{log.events.join(', ')}</td>
-                        <td>{log.gasUsed}</td>
-                        <td>{log.blockNumber}</td>
-                        <td>{log.status}</td>
-                        <td>{log.transactionHash}</td>
+                      <tr key={'log-' + index} className={log.error ? 'bg-danger' : ''}>
+                        <td className={log.error ? 'text-light' : ''}><DebugButton forceDebuggable transaction={{hash: log.transactionHash}}/></td>
+                        <td className={log.error ? 'text-light' : ''}>{`${log.name}.${log.functionName}(${log.paramString})`}</td>
+                        <td className={log.error ? 'text-light' : ''}>{log.events.join(', ')}</td>
+                        <td className={log.error ? 'text-light' : ''}>{log.gasUsed}</td>
+                        <td className={log.error ? 'text-light' : ''}>{log.blockNumber}</td>
+                        <td className={log.error ? 'text-light' : ''}>{log.status}</td>
+                        {log.transactionHash && <td className={log.error ? 'text-light' : ''}>{log.transactionHash}</td>}
+                        {log.error && <td className={log.error ? 'text-light' : ''}>{log.error}</td>}
```

Comment:
You could use `classnames` here, put the result in a variable and use the variable instead of doing the same ternary over and over.
https://github.com/JedWatson/classnames#usage

Reply:
Done.